lib/module_player.js

```diff
@@ -148,16 +148,16 @@ export function configure({makeEmptyModule, monitor, debug, time, logError}) {
       // shown.
       if (this.nextModule != module) {
         // Clean up the module and await further instructions.
-        debug(`Switching to ${this.nextModule.name} after prior preparation.`);
+        log(`Switching to ${this.nextModule.name} after prior preparation.`);
```

Comment:
gah.
So log is a function that you can call, which also has properties (log, error, debug) that are functions too?
Not sure if worth it, but willing to go along with it I guess. It's confusing to see both log() and log.error().

Reply:
Ha ha ha!

The goal here was to avoid changing the API for all of the modules, and because debug is a function, log had to be a function. However, I wanted to expose more shiny functionality to some users, hence, the keys on the function that you have an allergic reaction to.

Functions are just Objects, so they can have properties, too (e.g. myfunc.toString(), myfunc.arguments).

Reply:
Terrifying. Ok!